ghzdac/correction.py

```diff
@@ -1110,6 +1130,9 @@ def DACifyFT(self, signal, t0=0, n=8192, offset=0, nfft=None, loop=False,
         decayRates = np.array([x['GHz'] for x in self.decayRates])
         decayAmplitudes = self.decayAmplitudes
 
+        reflectionRates = np.array([x['GHz'] for x in self.reflectionRates])
```

Comment:
Not sure what it means that the server doesn't do units since the code has `x['GHz']`. Wherever `reflectionRates` is set perhaps we can make that a `ValueArray` instead of a list of `Value`, since that's what pylabrad unflattens `*v[Hz]` to anyway.